en/docs/consume/manage-subscription/advanced-topics/adding-an-api-subscription-deletion-workflow.md

```diff
@@ -0,0 +1,50 @@
+# Adding an API Subscription Deletion Workflow
+
+[Subscription Deletion]({{base_path}}/consume/manage-subscription/subscribe-to-an-api/) provides the capability to remove an already existing subscription. Attaching a custom workflow to the API subscription deletion, enables an admin to approve/reject the subscription deletion request made for an existing subscription. Note that only an admin is able to approve/reject a subscription deletion request.
+
```

Comment:
" Attaching a custom workflow to the API subscription deletion" >> Is this a custom workflow?